packages/stream_chat/lib/src/client/client.dart

```diff
@@ -327,7 +362,8 @@ class StreamChatClient {
         _ws.connectionStatusStream.skip(1).listen(_connectionStatusHandler);
 
     try {
-      return await _ws.connect(user);
+      await _ws.connect(user);
+      return state.user!;
```

Comment:
maybe?

```dart
      final event = await _ws.connect(user);
      return event.me.merge(ownUser);
```

Reply:
I think there's no need for this

Reply:
I just don't want to miss any user information

Reply:
yeah but state.user is updated using the events and it's the latest version in the backend right?